pom.xml

```diff
@@ -1,15 +1,19 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                             http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-    <groupId>net.wasdev.wlp.sample</groupId>
-    <artifactId>swagger-sample</artifactId>
-    <packaging>pom</packaging>
-    <version>1.0-SNAPSHOT</version>
-    <name>WAS Liberty Sample - Using Swagger in Liberty</name>
+    <parent>
+        <groupId>net.wasdev.wlp.maven.parent</groupId>
+        <artifactId>liberty-maven-app-parent</artifactId>
+        <version>2.0-SNAPSHOT</version>
+    </parent>
 
+	<artifactId>swagger-sample-application</artifactId>
```

Comment:
you will want to keep
```
<groupId>net.wasdev.wlp.sample</groupId>
<artifactId>swagger-sample</artifactId>
<version>1.0-SNAPSHOT</version>
```
otherwise it will inherit groupid and version from the parent.